sergey.md

```diff
@@ -0,0 +1,27 @@
+# Comments
+
```

Comment:
Also, still need:
- Update to Compute Changelog.md, 'Added PPG support to New-AzVM and New-AzVmss simple parameter sets'
- Markdown help updates (the syntax diagram has changed slightly
- Regression test (if time)